var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -29,19 +29,18 @@
 from spack import *
 
 
-class Rose(Package):
+class Rose(AutotoolsPackage):
     """A compiler infrastructure to build source-to-source program
        transformation and analysis tools.
        (Developed at Lawrence Livermore National Lab)"""
 
     homepage = "http://rosecompiler.org/"
-    url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
+    #url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
 
-    version('0.9.7', 'e14ce5250078df4b09f4f40559d46c75')
```

Comment:
We usually keep old versions of packages in case someone still needs them. See https://groups.google.com/d/msg/spack/afI3-EbiglM/HTVM1oX2EwAJ